db/migrate/20190102182816_create_versioned_asset_manager_image_variants.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class CreateVersionedAssetManagerImageVariants < ActiveRecord::Migration[5.2]
+  def change
+    create_table :versioned_asset_manager_image_variants do |t|
+      t.references :image_revision,
+                   foreign_key: { to_table: :versioned_image_revisions,
+                                  on_delete: :cascade },
+                   index: false,
+                   null: false
+
+      t.references :asset_manager_file,
+                   foreign_key: { to_table: :versioned_asset_manager_files,
+                                  on_delete: :restrict },
+                   index: false,
+                   null: false
+
+      t.string :variant, null: false
```

Comment:
I don't think size is the right word as I think there's more of a nuance than just size to each kind of file. I did want to avoid variant but I didn't have a better synonym so I'd be voting for variant over size but if we can think of something better that'd be good.

Reply:
Maybe you can explain then nuance ;-). I notice Wordpress does use this word for what I believe is the same thing.